modules/payloads/singles/linux/mipsbe/exec.rb

```diff
@@ -71,11 +70,8 @@ def generate(_opts = {})
     shellcode = shellcode + command_string + "\x00"
 
     # we need to align our shellcode to 4 bytes
-    while shellcode.bytesize%4 != 0
-      shellcode = shellcode + "\x00"
-    end
+    shellcode += "\x00" while shellcode.bytesize % 4 != 0
```

Comment:
nab: Might disable this rubocop rule in the future, it's OK here - but not sure if I'm a bit fan of it 😄

Reply:
It almost reads like a `do .. while` rather than a `while .. do`

Reply:
Yeah me neither. I generally don't like right side conditionals except for guard clauses.

This rule is [Style/WhileUntilModifier](https://docs.rubocop.org/rubocop/cops_style.html#stylewhileuntilmodifier). It is part of the Ruby style guide: https://rubystyle.guide/#while-as-a-modifier

I'm inclined to leave it enabled.